ignite/base/events_driven.py

```diff
@@ -62,17 +64,20 @@ def __init__(self) -> None:
         self.logger = logging.getLogger(__name__ + "." + self.__class__.__name__)
 
     def register_events(
-        self, *event_names: Union[List[str], List[EventEnum]], event_to_attr: Optional[dict] = None,
+        self, *event_names: Union[List[str], List[EventEnum]], attr_to_events: Optional[dict] = None,
```

Comment:
I think we have to move `attr_to_events` somewhere else. Here, it is unused and does not make much sense.
Maybe, we'll have to add *kwargs to make mypy happy while inheriting from this class and overridding the method

Reply:
Oh, I see, thanks for pointing to this